include/nil/blueprint/assigner.hpp

```diff
@@ -1875,6 +1821,7 @@ namespace nil {
             generation_mode gen_mode;
             std::vector<branch_desc> curr_branch;
             component_calls statistics;
+            std::vector<typename BlueprintFieldType::value_type> internal_storage;
```

Comment:
It would be nice to have this field documented with some notes about what is it and how it works